python/langsmith/_internal/_background_thread.py

```diff
@@ -168,6 +168,30 @@ def _tracing_thread_handle_batch(
             tracing_queue.task_done()
 
 
+def _otel_tracing_thread_handle_batch(
+    client: Client,
+    tracing_queue: Queue,
+    batch: List[TracingQueueItem],
+) -> None:
+    """Handle a batch of tracing queue items by exporting them to OTEL."""
+    try:
+        ops = combine_serialized_queue_operations([item.item for item in batch])
+
+        run_ops = [op for op in ops if isinstance(op, SerializedRunOperation)]
+
+        if run_ops and client.otel_exporter is not None:
+            client.otel_exporter.export_batch(run_ops)
```

Comment:
maybe log something here because `client.otel_exporter` should never be `None` here?